rules/table-of-contents.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const visit = require('unist-util-visit');
+const chalk = require('chalk');
+
+module.exports = (ast, file, language, done) => {
+	visit(ast, 'heading', (node, index) => {
+		const title = node.children[0].value;
+
+		if (index === 1 && title !== 'Contents') {
+			file.warn(`The first section must be ${chalk.bold('Contents')}`);
+			return false;
+		}
+	});
+	done();
```

Comment:
No need for `done()`. Just bump `remark-lint` to `^4.1.0` in package.json. See https://github.com/wooorm/remark-lint/commit/8dedd190c1cd78867cf6797cc3e4b284ff993556